yarn-project/bot/src/factory.ts

```diff
@@ -410,15 +410,15 @@ export class BotFactory {
     const mintAmount = await portal.getTokenManager().getMintAmount();
     const claim = await portal.bridgeTokensPublic(recipient, mintAmount, true /* mint */);
 
-    const isSynced = async () => await this.pxe.isL1ToL2MessageSynced(Fr.fromHexString(claim.messageHash));
-    await retryUntil(isSynced, `message ${claim.messageHash} sync`, this.config.l1ToL2MessageTimeoutSeconds, 1);
+    await this.withNoMinTxsPerBlock(() =>
+      waitForL1ToL2MessageReady(this.pxe, Fr.fromHexString(claim.messageHash), {
+        timeoutSeconds: this.config.l1ToL2MessageTimeoutSeconds,
+        forPublicConsumption: true,
```

Comment:
```suggestion
        forPublicConsumption: false,
```

My mistake, this is wrong on the original. The cross-chain `claim` on fee juice runs in private-land, even though the balance is increased in public-land.

Reply:
And the fact that this didn't trigger CI means I'm missing a test.